website/cue/reference/components/transforms/route.cue

```diff
@@ -32,7 +32,8 @@ components: transforms: route: {
 			description: """
 				A table of route identifiers to logical conditions representing the filter of the route. Each route
 				can then be referenced as an input by other components with the name `<transform_name>.<route_id>`.
-				Note, `_default` is a reserved output name and cannot be used as a route name.
+				If an event doesn't match any route, it will be sent to the `<transform_name>._unmatched` output.
+				Note, `_default` and `_unmatched` are reserved output names and cannot be used as route names.
```

Comment:
The `_default` reservation is enforced during config validation: https://github.com/vectordotdev/vector/blob/master/src/config/validation.rs#L146. It seems like we could extend `check_outputs` to also check for `_unmatched` (but scoped only to the `route` transform).